packages/node/src/executors/execute/execute.impl.spec.ts

```diff
@@ -215,6 +220,10 @@ describe('NodeExecuteBuilder', () => {
   });
 
   it('should log errors from killing the process', async () => {
+    (devkit.runExecutor as any).mockImplementation(function* () {
+      yield { success: true, outfile: 'outfile.js' };
+      yield { success: true, outfile: 'outfile.js' };
+    });
```

Comment:
The old test cases for this spec and the one below were non-deterministic:

The state of `subProcess` was managed as a file-level global previously. These tests would only pass when at least one previous test had been run to populate the `subProcess` var.

With that subProcess state encapsulated, there was no way to make this test pass without altering it.